README.md

```diff
@@ -3,36 +3,130 @@
+  * [2. Installing the resource aggregator for ODIM, the Generic redfish plugin (GRF), and the Unmanaged Rack Plugin (URP)](#2-installing-the-resource-aggregator-for-odim-the-generic-redfish-plugin-grf-and-the-unmanaged-rack-plugin-urp)
+    + [Default user credentials for ODIMRA, the GRF Plugin, and URP](#default-user-credentials-for-odimra-the-grf-plugin-and-urp)
 - [Modifying default configuration parameters for the resource aggregator](#modifying-default-configuration-parameters-for-the-resource-aggregator)
 - [Configuring proxy for Docker](#configuring-proxy-for-docker)
+- [Uninstalling ODIMRA](#uninstalling-odimra)
+- [CI Process](#ci-process)
+  * [GitHub action workflow details](#github-action-workflow-details)
+
+
+
+# Introduction 
+
+ Welcome to Resource Aggregator for Open Distributed Infrastructure Management!
+
+Resource Aggregator for Open Distributed Infrastructure Management (ODIMRA) is a modular, open framework for
+simplified management and orchestration of distributed physical infrastructure. It provides a unified management platform for
+converging multivendor hardware equipment. By exposing a standards-based programming interface, it enables easy and
+secure management of wide range of multivendor IT infrastructure distributed across multiple data centers.
+
+ODIMRA framework comprises the following two components.
+
+- The resource aggregation function (the resource aggregator)
+
+  The resource aggregation function is the single point of contact between the northbound clients and the
+  southbound infrastructure. Its primary function is to build and maintain a central resource inventory. It exposes
+  Redfish-compliant APIs to allow northbound infrastructure management systems to:
+    - Get a unified view of the southbound compute, local storage, and Ethernet switch fabrics available in the
+      resource inventory.
+    - Gather crucial configuration information about southbound resources.
+    - Manipulate groups of resources in a single action.
+    - Listen to similar events from multiple southbound resources.
+	
+- One or more plugins
+
+  The plugins abstract, translate, and expose southbound resource information to the resource aggregator through
+  RESTful APIs. HPE Resource Aggregator for ODIM supports:
+ 
+    - Generic Redfish plugin for ODIM (GRF): Generic Redfish plugin that can be used as a plugin for any Redfishcompliant
+      device.
+	- Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks. 
+    - Integration of additional third-party plugins.  
+
+
+
+
+##  Resource Aggregator for ODIM logical architecture
+
+Resource Aggregator for ODIM framework adopts a layered architecture and has many functional layers.
+
+
+The following figure shows these functional layers of Resource Aggregator for ODIM deployed in a data center.
+
+![ODIM_architecture](images/arch.png)
```

Comment:
This link is now working.

Reply:
It will work once it is merged to this branch.